scripts/benchmarks.sh

```diff
@@ -35,7 +35,7 @@ do
 		--extrinsic='*' \
 		--steps=$steps  \
 		--repeat=$repeat \
-		--json \
+		--json-file=./bench-statemine.json \
```

Comment:
If you do not plan on analyzing the resulting json files, you can also kick it out completely.

Reply:
They won't take any time to write out and if you did want to have a look at them afterwards you don't really want to re-run the benchmarks as this could take a long time.